apps/nova/src/components/Layout/Sidebar.tsx

```diff
@@ -145,12 +167,24 @@ const Sidebar = () => {
           {intl("sidebar.settings")}
         </Typography>
         <ul className="flex flex-col gap-2">
-          <SidebarItem
-            to={Route.Settings}
-            Icon={Settings}
-            label={intl("sidebar.settings")}
-            active={location.pathname === Route.Settings}
-          />
+          {user?.role === IUser.Role.Tutor ? (
```

Comment:
Remember that we have two types of tutors (Tutors and tutor managers). The logic here doesn't not take into consideration the tutor manger.